### ISSUE TITLE
ReferenceError: React is not defined

### ISSUE THREAD

**Issue by NadsonKT** [apexcharts/react-apexcharts#642]
I’m getting this error after the update:

```
react-apexcharts.js?v=be36e25b:8397 Uncaught ReferenceError: React is not defined
    at Charts (react-apexcharts.js?v=be36e25b:8397:7)
    at renderWithHooks (chunk-67DEV5DB.js?v=826a3772:11548:26)
    at mountIndeterminateComponent (chunk-67DEV5DB.js?v=826a3772:14926:21)
    at beginWork (chunk-67DEV5DB.js?v=826a3772:15914:22)
    at HTMLUnknownElement.callCallback2 (chunk-67DEV5DB.js?v=826a3772:3674:22)
    at Object.invokeGuardedCallbackDev (chunk-67DEV5DB.js?v=826a3772:3699:24)
    at invokeGuardedCallback (chunk-67DEV5DB.js?v=826a3772:3733:39)
    at beginWork$1 (chunk-67DEV5DB.js?v=826a3772:19765:15)
    at performUnitOfWork (chunk-67DEV5DB.js?v=826a3772:19198:20)
    at workLoopSync (chunk-67DEV5DB.js?v=826a3772:19137:13)
```

**Comment by AhmadAymanA99** [comment]
same error with me thought I didn't run any upgrade 

version:
"apexcharts": "^3.53.0",
"react-apexcharts": "^1.4.1",

when just deleted the node_modules and installed them again.

```
Uncaught ReferenceError: React is not defined
    at Charts (react-apexcharts.min.js:1:1)
    at renderWithHooks (react-dom.development.js:15486:1)
    at mountIndeterminateComponent (react-dom.development.js:20103:1)
    at beginWork (react-dom.development.js:21626:1)
    at beginWork$1 (react-dom.development.js:27465:1)
    at performUnitOfWork (react-dom.development.js:26596:1)
    at workLoopSync (react-dom.development.js:26505:1)
    at renderRootSync (react-dom.development.js:26473:1)
    at recoverFromConcurrentError (react-dom.development.js:25889:1)
    at performSyncWorkOnRoot (react-dom.development.js:26135:1)
```

**Comment by heyjmac** [comment]
Same error!

**Comment by junedchhipa** [comment]
Released `react-apexcharts@1.4.4` which fixes the error.
Please upgrade and report here whether it fixed the original issue.

**Comment by AhmadAymanA99** [comment]
Yes, that fixed the original issue with me thanks
